nifi-commons/nifi-properties/src/main/java/org/apache/nifi/util/NiFiProperties.java

```diff
@@ -71,6 +71,7 @@
     public static final String ADMINISTRATIVE_YIELD_DURATION = "nifi.administrative.yield.duration";
     public static final String PERSISTENT_STATE_DIRECTORY = "nifi.persistent.state.directory";
     public static final String BORED_YIELD_DURATION = "nifi.bored.yield.duration";
+    public static final String PROCESSOR_START_TIMEOUT = "nifi.processor.start.timeout";
```

Comment:
I don't think we need to add default since it's handled during retrieval (basically if property is null or doesn't exist). 
I'll change to string based representation of time although I am still struggling with the whole concept and if indeed it is more user friendly.